src/test/java/stepdefinitions/UserProfile.java

```diff
@@ -0,0 +1,109 @@
+package stepdefinitions;
+
+import java.util.concurrent.TimeUnit;
+import Base.BaseActions;
+
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import context.TestContext;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import pageobject.HomePage;
+import pageobject.LoginPage;
+import pageobject.UserProfilePage;
+import sun.java2d.cmm.Profile;
+import managers.PageObjectManager;
+
+public class UserProfile 
+
+{
+	private PageObjectManager objectmanager;
+	private TestContext context;
+	private LoginPage loginPage;
+	private HomePage homePage;
+	private UserProfilePage profile;
+	private BaseActions base;
+	
+	WebDriver driver;
+	WebDriverWait wait;
```

Comment:
Wait is also not needed here as it is already written for page elements

Reply:
It was added by mistake. Removed it.